Library/Formula/jenkins.rb

```diff
@@ -58,12 +64,12 @@ def caveats; <<-EOS.undent
   test do
     ENV["JENKINS_HOME"] = testpath
     pid = fork do
-      exec "#{bin}/jenkins"
+      exec "java -Djava.io.tmpdir=#{testpath} -jar #{opt_libexec}/jenkins.war"
```

Comment:
@dunn Ok I added the followin to the test block as per your suggestion:

```ruby
ENV["_JAVA_OPTIONS"] = "-Djava.io.tmpdir=#{testpath}"
```
Also, the SHA256 of http://mirrors.jenkins-ci.org/war-rc/2.0/jenkins.war seems to have remained stable for the past week, so I'm optimistic it will remain that way.